kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Environment.java

```diff
@@ -34,8 +34,11 @@ private Environment() {
      * Env. variables names
      */
     private static final String KAFKA_VERSION_ENV = "KAFKA_VERSION";
+    private static final String KROXY_OPERATOR_IMAGE_ENV = "KROXYLICIOUS_OPERATOR_IMAGE_NAME";
```

Comment:
nit: can we use the full name KROXYLICIOUS rather than KROXY. The name Kroxy belongs to someone else.

Reply:
sure, I'll change that